windows/export_vm/export_vm.yml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Description:
+#   This playbook is used for exporting VM to OVF
+# template, by default the exported template files
+# will be saved in '/tmp/' folder.
+#
+- name: export_ovf
+  hosts: localhost
+  gather_facts: no
+  vars_files:
+    - "{{ testing_vars_file | default('../../vars/test.yml') }}"
+  tasks:
+    # Check VM power status
+    - include_tasks: ../../common/vm_get_power_state.yml
+    # Power off VM if it's not in poweredOff state
+    - include_tasks: ../utils/shutdown_vm.yml
+    # Remove other CDROM devices except 'CD/DVD drive 1'
+    - include_tasks: ../../common/vm_get_cdrom_devices.yml
+    - include_tasks: ../../common/vm_configure_cdrom.yml
+      vars:
+        cdrom_state: absent
+        cdrom_controller_type: "{{ item.controller_label.split(' ')[0] | lower }}"
+        cdrom_controller_num: "{{ item.bus_num }}"
+        cdrom_unit_num: "{{ item.unit_num }}"
+      when: item.label != "CD/DVD drive 1"
+      with_items: "{{ cdrom_device_list }}"
+      when:
+        - cdrom_device_list is defined
+        - cdrom_device_list | length > 1
```

Comment:
We might need to consider a scenario: if the VM is exported from an existing VM which still have CD/DVD drive1 connecting to an ISO image, we still need to reconfigure the CD/DVD drive 1 to a client device.

Reply:
Yeah, right, added it.